docs/releasing/publishing-from-a-support-branch.md

```diff
@@ -0,0 +1,157 @@
+# Publish GOV.UK Frontend when `main` branch has unreleasable changes
+
+This document is for GOV.UK Design System developers who need to publish a support branch of GOV.UK Frontend when the `main` branch contains unreleasable changes. For example, you might need to release a fix as part of a:
+
+- patch release, after the team has started to merge changes for a new feature release into the `main` branch - for example, a 3.14.x release once we've started merging changes for 3.15.0
+- release, after the team has started to merge changes for a new breaking release into the `main` branch - for example, a 3x.x release once we've started merging changes for 4.0.0
+
+Do not use this document to release changes for previous major releases. We have not tested the document in this scenario, and extra work would be needed to tell NPM not to mark the release as the 'latest'.
+
+If you want to publish the `main` branch for GOV.UK Frontend, [follow the steps in Publishing GOV.UK Frontend](/docs/releasing/publishing.md).
+
+If the `main` branch only has a few unreleasable changes, you can temporarily revert these changes.
+
+1. Revert the unreleasable changes on the `main` branch.
+2. Publish GOV.UK Frontend.
+3. Add the reverted changes back into the `main` branch.
+
+However, this approach has risks. For example, it creates a messy commit history on the `main` branch.
+
+## Publish GOV.UK Frontend from a support branch
+
```

Comment:
Should there be some content between the headings?

Reply:
It feels unnatural, but I don't think so...Is there anything we could add that wouldn't be a reiteration of the title?

This is something that crops up in some other Design System content - like in our [release notes](https://github.com/alphagov/govuk-frontend/releases/tag/v3.14.0) (no content between 'New features' and the first h3 heading) and our [Frontend API docs](https://frontend.design-system.service.gov.uk/sass-api-reference/#sass-api-reference).

There are definitely times when I've thought we needed this sort of content, but I haven't found anything in the style guide or on WCAG to back that up...

Reply:
As discussed with @cmcnallygds, we'll revisit the headings issue after we've researched a bit more about recommended practice to benefit users.